tests/TestContract.sol

```diff
@@ -4,6 +4,14 @@ contract TestContract {
     function foo(uint16 a, uint8 b, uint32 c) returns (uint64) {
         return a +b + c;
     }
+    function function_int128(uint64 a, uint64 b) public view returns (uint128) {
+        uint128 x = a + b;
+        return x;
+    }
+    function function_int256(uint64 a, uint64 b, uint64 c, uint64 d) public view returns (uint256) {
```

Comment:
atm the code is just testing encoding of the parameters to the transaction data. Could you please change the method to something like:

`function function_int256(uint64 a, uint128 b, uint256) public view returns (uint256)`

you can remove the `function_int128` method

(test for the method return value will be useful once we implement transaction decoding)

Reply:
Got it. It seems C++ in general can't handle a uint128/uint256 string constant in a variable say for example 

`uint128_t val = 0x00FF00FF00FF00FF 00FF00FF 00FF00FF \\ This constant cannot be represented`